### ISSUE TITLE
Default settings for  maxNumberProblems per topic and maxTime are too low

### ISSUE THREAD

**Issue by IvonArroyo** [marshall62/mathspring#301]
David, Rafael,

The default settings for  maxNumberProblems per topic and maxTime are too low, they should be set to something like 15 problems max, and 15 minutes max.

Worse, there is no way to change them any more from the teacher tools, not even with the administrator user (no submit button).

--Ivon

**Comment by IvonArroyo** [comment]
I changed them for this study now directly on the DB.

**Comment by marshall62** [comment]
Yes.  We need quite a few additions to the teacher tools to allow us to
edit the parts of the pedagogy and the new rules for controlling the
companions.  It's on my list but its a fairly substantial project.
